testing/ember-app/tests/tests/integration/usage-test.ts

```diff
@@ -264,6 +264,29 @@ module('Usage', function (hooks) {
     assert.strictEqual(context.numCalled, 13);
   });
 
+  test('merging passed context by default', async function (assert) {
+    let toggle = createMachine({
+      initial: 'inactive',
+      context: { foo: 'foo' },
+      states: {
+        inactive: { on: { TOGGLE: 'active' } },
+        active: { on: { TOGGLE: 'inactive' } },
+      },
+    });
+
+    this.owner.register('component:toggle-machine', toggle);
```

Comment:
instead of register, can all new tests instead do: `<this.toggle>`

like:
```js
this.setProperties({ toggle, context: { ...}  });

hbs`
  <this.toggle @context=...
`
```